src/handlers/entity/hatch.js

```diff
@@ -109,16 +109,16 @@ export const process = (tuples) => {
           break
         case 42:
           {
-            if (isPolyline) loop.bulge = value
+            if (isPolyline) loop.entities[0].vertices[loop.entities[0].vertices.length-1].bulge = value
             else fillDrawEntity(type, drawType, parseFloat(value))
           }
           break
         case 72:
           {
             // !Polyline --> 1 = Line; 2 = Circular arc; 3 = Elliptic arc; 4 = Spline
-            // Polyline -->  bulge
+            // Polyline -->  hasBulge
             drawType = parseFloat(value)
-            loop[isPolyline ? 'bulge' : 'edgeType'] = drawType
+            loop[isPolyline ? 'hasBulge' : 'edgeType'] = drawType
```

Comment:
@ieskudero Could you elaborate on the (breaking change?) rename of `bulge` to `hasBulge` on row 121, please? Also in relation to the `bulge` property on row 112. Not sure I understand why this is needed.

Reply:
About the points->controlPoints renaming: it is in fact a breaking change, sorry. I did it to be more consistent with the names of properties, since spline has control points, not points.
About the bulge->hasBulge renaming: After realizing the error I understood the property, so I changed the name accordingly. I don't think is necessary (unless there are bulges with 0 value) but I put it anyway.

I can set a different pull request separating them both, if you want.

Reply:
Sorry about the delay in answering here, lots to do! I'm still a bit confused about the loop.hasBulge property, looks to me like it will be a number (parseFloat on line 120) but the name signals a boolean. Would be great if you could help me understand @ieskudero

Reply:
According to the dxf documentation ([Boundary path data](https://documentation.help/AutoCAD-DXF/WS1a9193826455f5ff18cb41610ec0a2e719-7a11.htm#WSc30cd3d5faa8f6d8e3b895ffb7556c21-7fec)) has bulge is a flag. I assume it is a true/false flag, since it doesn't mark it as a bit coded flag. I could check some examples to be sure but I bet it is a 0-1 value flag. And the parseFloat must be a mistake, inherited from the previous assumption that this was a bulge value instead of a bulge flag, it can be changed to parse to boolean or integer.

Reply:
I have read this several times. The name change for points to vertices is not necessary.

And the bulge change doesn't make sense. It looks like a fix for a special case.

maybe this needs more,test cases?

Reply:
I didn't mean to create such a long argument with this pull request, sorry. I am thinking that maybe we can take back this pull request and do other pull requests that are more focused and with less changes, because I think we are mixing things here. Then we can discuss each problem separately:
1.- renaming of properties(points to control points, points to vertices)
2.- bulge property fix
3.- renaming of bulge to hasBulge

Reply:
@skymakerolof, any opinion about this?

Reply:
I agree that the ` loop.entities[0].vertices[loop.entities[0].vertices.length-1].bulge` part looks like it is handling some specific case? And that we need more test cases. Also like you say, once change per pull request would make things easier to follow and discuss!

Reply:
OK, perfect, you can close this one and I will make them separately